MTA10/mods/shared_logic/CClientObject.cpp

```diff
@@ -502,16 +510,14 @@ void CClientObject::Create ( void )
                 // Add XRef
                 g_pClientGame->GetGameEntityXRefManager ()->AddEntityXRef ( this, m_pObject );
 
-                // If set to true,this has the effect of forcing the object to be static at all times
-                m_pObject->SetStaticWaitingForCollision ( m_bIsStatic );

```

Comment:
I replaced this with SetFrozen below. Generally this _StaticWaitingForCollision_ flag has effect only for _true_ so recreating object was required.

If you are worried about this initial freeze until something touch object - it's still here.